app.py

```diff
@@ -1,17 +1,20 @@
 #!/usr/bin/env python3
 import os
 import boto3 
+import string
+import random
 import aws_cdk as cdk
 import aws_cdk.assertions as assertions
 from sds_in_a_box.sds_in_a_box_stack import SdsInABoxStack
 
+### Configuration variables
+# A random unqiue ID for this particular instance of the SDS
+SDS_ID = "".join( [random.choice(string.ascii_lowercase) for i in range(8)] )
```

Comment:
Yes I would say definitely so!  I just got somewhat lazy this ticket and kind of skipped out on some of quality-of-life tools.  

One thing I was wondering about was maybe the configuration might live in another type of configuration file (or even the same config.json file as the file types), rather than asking the user for prompts? Or maybe it asks the user for prompts if it can't find an initial config file with an ID, intial email, instance types, etc.   

That said, I think maybe adding these more "user-friendly" components might be better suited to seperate ticket, especially since this one contains some pretty large changes as it is?

Reply:
I like the configuration file idea. Would a configuration file check happen in app.py or is there another place where that makes more sense? And I agree, that wouldn't be something for this ticket.

Reply:
We could try it in the app.py file, and see if it runs when we call "cdk synth" or "cdk deploy".  But if that doesn't work, we could make a separate script that checks for a config file and/or asks for user input, and then calls cdk synth/cdk deploy for the user.